v1/src/main/java/com/google/cloud/teleport/spanner/TextImportPipeline.java

```diff
@@ -212,12 +202,10 @@ public interface Options extends PipelineOptions {
         order = 10,
         optional = true,
         description = "Date format",
-        helpText =
-            "The format used to parse date columns. By default, the pipeline tries to parse the"
-                + " date columns as \"yyyy-MM-dd[' 00:00:00']\" (e.g., 2019-01-31, or 2019-01-31"
-                + " 00:00:00). If your data format is different, please specify the format using"
-                + " the java.time.format.DateTimeFormatter patterns. For more details, please refer"
-                + " to https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/format/DateTimeFormatter.html")
+        helpText =  "The format used to parse date columns. By default, the pipeline tries to parse the date columns "
+               + "as `yyyy-M-d[' 00:00:00']`, for example, as 2019-01-31 or 2019-1-1 00:00:00. If your date format "
+               + "is different, specify the format using the [java.time.format.DateTimeFormatter]"
```

Comment:
I don't think markdown will render in all contexts, so probably better to leave this one `java.time.format.DateTimeFormatter (https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/format/DateTimeFormatter.html)` instead of `[java.time.format.DateTimeFormatter](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/format/DateTimeFormatter.html)`